GeosCore/hco_utilities_gc_mod.F90

```diff
@@ -1706,39 +1712,35 @@ SUBROUTINE Get_GC_Restart( Input_Opt, State_Chm, State_Grid, &
          SpcInfo%Is_InRestart = .FALSE.
       ENDIF
 
-      ! If data is in file, read in as [mol/mol] and convert to
-      ! [kg/kg dry]. Otherwise, set to background value [mol/mol]
-      ! either stored in species database (advected species all levels and
-      ! non-advected species levels in the chemistry grid) or a small number
-      ! (non-advected species levels above the chemistry grid) converted to
-      ! [kg/kg dry]
+      ! If data is in file, read in as [mol/mol]. Otherwise, set to
+      ! background value [mol/mol] either stored in species database
+      ! (advected species all levels and non-advected species levels
+      ! in the chemistry grid) or a small number (non-advected species
+      ! levels above the chemistry grid).
       IF ( SpcInfo%Is_InRestart ) THEN
 
-         ! Print the min & max of each species as it is read from
+         ! Print the min, max, and sum of each species as it is read from
          ! the restart file in mol/mol
          IF ( Input_Opt%amIRoot ) THEN
-            WRITE( 6, 120 ) N, TRIM( SpcInfo%Name ), &
+            WRITE( 6, 530 ) N, TRIM( SpcInfo%Name ), &
```

Comment:
I think we can simply replace `SUM ( Ptr3D(:,:,1:State_Grid%NZ) )` with `SUM( Ptr3D )` since the `Temp3D` array is defined with NZ levels.

Reply:
Done!